### PR TITLE
xarray-tutorial improvements

### PR DESCRIPTION
**Pull request by dcherian**
@scottyhq @JessicaS11 

I got started on an outline for a blogpost about the upgrades to the Xarray tutorial website. You should have rights to just push to the branch.

### CODE REVIEW

**Comment by TomNicholas** [comment]
Could we perhaps revive this idea for SciPy 2024? 😁 The tutorial site still doesn't get the fanfare it deserves!

**Comment by dcherian** [comment]
We should publish it before that. Mind pushing some text?

**Comment by dcherian** [comment]
@scottyhq @JessicaS11 shall we try and finish this and use it to advertise the upcoming tutorial?

File: src/posts/xarray-tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+<img
+  src='/posts/tutorial/tutorial-before-after.png'
+  alt='Tutorial outline before/after'
+  width='60%'
+/>
+
+We've been particularly happy with the [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst) extension, which extends the basic Markdown rendering with in JupyterLab and ensures Notebooks rendered as a static webpage look identical to an exectuable notebook in the JupyterLab interface.
+
+Finally, we think the Xarray Tutorial is a great way to engage new contributors to Open Source Sofware Development. Every Jupyterbook webpage has a GitHub icon dropdown in the upper right hand corner with a 'Open Issue' and 'Suggest Edit' button. This really simplifies the process of community development and we love that during workshops people totally new to the library have made improvements to the content as we run through it. The repository now has over 30 contributors!
```
Comment (TomNicholas) [inline src/posts/xarray-tutorial/index.md:50]: *Software

File: src/posts/xarray-tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## New and improved material
+
+In addition to redesigning the layout, we've improved existing material and added a lot of new material focused on intermediate and advanced concepts:
+
+1. Revamped fundamentals
+
+- Data structures, indexing, alignment & broadcasting
+
+1. All New Intermediate level material
```
Comment (TomNicholas) [inline src/posts/xarray-tutorial/index.md:60]:
```suggestion
2. All New Intermediate level material
```

**Review by TomNicholas — COMMENTED** [review]
Nice @scottyhq !

File: src/posts/xarray-tutorial/index.md (hunk 1 above)
Comment (dcherian) [inline src/posts/xarray-tutorial/index.md:50]:
```suggestion
Finally, we think the Xarray Tutorial is a great way to engage new contributors to Open Source Software Development. Every Jupyterbook webpage has a GitHub icon dropdown in the upper right hand corner with a 'Open Issue' and 'Suggest Edit' button. This really simplifies the process of community development and we love that during workshops people totally new to the library have made improvements to the content as we run through it. The repository now has over 30 contributors!
```

File: src/posts/xarray-tutorial/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Approach
+
+The Xarray Tutorial Website https://tutorial.xarray.org hosts a curated selection of Jupyter Notebooks that illustrate concepts, common usage patterns, and long-form examples of using Xarray for data analytics.
+
+We reworked existing material to bite-sized chunks that can be remixed as needed. For example, rather than organizing material by specific workshops at a top-level, we now organize by topic and then build a learning path for each event. This way future learners can benefit from a guided tour through the material.
+For example see the [Fundamental](https://tutorial.xarray.dev/overview/fundamental-path/README.html) and [Intermediate](https://tutorial.xarray.dev/overview/intermediate-path/README.html) learning paths that were presented at SciPy 2022 and SciPy 2023 respectively.
+
```
Comment (dcherian) [inline src/posts/xarray-tutorial/index.md:35]:
```suggestion
We believe this method of organizing material also lends it self to easy reuse for other tutorial/workshop activities that may not want to work through material in the manner the canonical ordering does. 
```

or something like that?

**Comment by dcherian** [comment]
Good to go, @scottyhq @JessicaS11  @negin513

**Review by scottyhq — APPROVED** [review]
Looks good to me! Could spruce it up with more screenshots, but I like that it's short and is another way of advertising the upcoming scipy2024 workshop

**Comment by TomNicholas** [comment]
FYI

https://x.com/DahnJahn/status/1800563901664563710

**Pull request merged**